samples/HybridApp/HybridApp/WebUI/Pages/Index.razor

```diff
@@ -1,7 +1,14 @@
 ﻿@page "/"
+@inject NavigationManager NavigationManager
 
 <h1>Hello, world!</h1>
 
 Welcome to your new app.
 
 <SurveyPrompt Title="How is Blazor working for you?" />
+
+<a href="https://gaohaoyang.github.io/history-pjax-demo/">History demo</a>
+
+<button @onclick="@( () => this.NavigationManager.NavigateTo("", true))"> Force Reload </button>
+
+<button @onclick="@( () => this.NavigationManager.NavigateTo("/counter", true))"> Force reload to counter </button>
```

Comment:
BTW @Eilon, I've kept these buttons in so that you can test. You might want to remove those (and the navigation manager above) from this file on final merge.

Reply:
Yup I'll remove these.